cli/src/base/AdvancedTransactionsCommandBase.ts

```diff
@@ -154,14 +160,17 @@ export default abstract class AdvancedTransactionsCommandBase extends AccountsCo
     timepointIndex: number | undefined,
     others: string | undefined,
     call: string,
-    maxWeight: number
+    maxWeight: {
+      refTime: string | number
+      proofSize: string | number
+    }
   ): Promise<MultisigAsMulti> {
     if (input) {
       const args = await this.getAsMultiInputFromFile(input)
       const otherSignatories = args.otherSignatories as string[]
       const otherSignatoriesSorted = sortAddresses(otherSignatories, JOYSTREAM_ADDRESS_PREFIX)
       args.otherSignatories = otherSignatoriesSorted
-      const maxWeightChanged = maxWeight !== (args.maxWeight as number)
+      const maxWeightChanged = maxWeight
```

Comment:
I interpret `maxWeightChanged` is a boolean but you are assigning it the actual weight value here.